src/main/scala/io/iohk/ethereum/db/dataSource/RocksDbDataSource.scala

```diff
@@ -175,12 +179,13 @@ class RocksDbDataSource(
       dbOptions.close()
       // 3. Free column families options
       cfOptions.close()
+      logger.info(s"DataSource closed successfully in the path: ${rocksDbConfig.path}")
     } catch {
       case NonFatal(e) =>
         logger.error("Not closed the DataSource properly, cause: {}", e)
```

Comment:
At least as a temporal solution i think is preferable to have some kind of domain modelling for `DataSource` runtime exceptions, instead of just log and throw exception, in that case we could just throw our custom exceptions that would provide better context about the actual issues and avoid the log